### PR TITLE
Added missing CSS3 and HTML5 icons for dropdown option list

### PR DESCRIPTION
**Pull request by arvislacis**
Fixes https://github.com/wintercms/wn-builder-plugin/issues/6

### CODE REVIEW

**Comment by LukeTowers** [comment]
All of these classes should probably also be updated to reference wn-icon instead of oc-icon, would you be willing to make that change and test it in this PR as well?

**Comment by arvislacis** [comment]
Yeah, I will try to look into that and replace oc-icon references to wn-icon.

**Comment by arvislacis** [comment]
@LukeTowers Done.

Should check https://github.com/wintercms/wn-builder-plugin/pull/8/commits/d7344a4278d685ee407cab71b2cfaffc16856f90#diff-0d9b1cafb1dd3d44397cc3b50e731dd12f6ae7e1bf9078295b8902f7faeb317dR109 - I added this for compability, otherwise previously set icon is not shown in the existing plugin settings. Maybe this `preg_replace()` function should be placed somewhere else.

Additionally added small update into en/lang.php as we discussed in https://github.com/wintercms/winter/discussions/228

File: lang/en/lang.php [hunk 1]
```diff
@@ -645,7 +645,7 @@
     ],
     'common' => [
         'error_file_exists' => "File already exists: ':path'.",
-        'field_icon_description' => 'Winter CMS uses Font Autumn icons: https://wintercms.com/docs/ui/icon',
+        'field_icon_description' => 'Winter CMS uses wrapper for Font Awesome icons: https://wintercms.com/docs/ui/icon',
```
Comment (LukeTowers) [inline lang/en/lang.php:648]:
```suggestion
        'field_icon_description' => 'Winter CMS uses a wrapper around Font Awesome icons: https://wintercms.com/docs/ui/icon',
```

**Comment by LukeTowers** [comment]
LGTM, will let @bennothommo merge it since he handles releases for this plugin

**Comment by bennothommo** [comment]
Thanks for this @arvislacis!

**Pull request merged**